volatility3/framework/plugins/windows/dumpfiles.py

```diff
@@ -243,6 +258,14 @@ def _generator(self, procs: List, offsets: List):
                         obj_type = entry.get_object_type(type_map, cookie)
                         if obj_type == "File":
                             file_obj = entry.Body.cast("_FILE_OBJECT")
+
+                            if self.config["regex"]:
```

Comment:
Presumably this could/should just check whether file_re is set?  Going back to the config allows the (admittedly, extremely unlikely) possibility that something changes the config between when you checked it and when you check it again.

Reply:
Yes this makes sense! I will change in both spots.